### ISSUE TITLE
[Enhancement] YTETA Tweak by iCrazeiOS

### ISSUE THREAD

**Issue by arichorn** [qnblackcat/uYouPlus#289]
## ⚠️ Have you read the [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ)? Make sure you visit the FAQ page first!!! ⚠️

_Make sure you complete the template. Otherwise, it will be closed without further explanation!_

### Device info
- iOS/iPadOS version: 15.6
- Device model: iPad Pro 10.5
- Sideload tool (Ex: AltStore, Sideloadly...)
- Version of uYouPlus: 2.1-16.42.3 + 2.1-17.26.2

### Disclaimer
Also I already have a feeling that this could have already been added in uYouPlus or CercubePlus and I wouldn’t have none but im taking a chance

### Describe the tweak
YTETA Sees exactly when a YouTube video will be done playing on the video player.

### Direct Link to it
https://github.com/iCrazeiOS/YTETA

**Comment by qnblackcat** [comment]
See https://github.com/qnblackcat/uYouPlus/pull/241

**Comment by arichorn** [comment]
I didn’t know you already had one added because I checked the readme.md file and couldn’t see any of those hidden or unlisted yt tweaks.

**Comment by burritosoftware** [comment]
@qnblackcat I would like to suggest for this to be reopened: The original closure reason was YTTrueTime, but that tweak displays the new adjusted playback time for playback speed (like 8:00 -> 4:00), whereas YTETA displays real-world time (like 9:41 PM) as to when a video will end. YTETA is one of the tweaks I used to use often while jailbroken until I switched to solely sideloading so I'd love to see it appear in uYouPlus. Thanks!
